app/domain/content/query.rb

```diff
@@ -163,7 +163,7 @@ def set(instance_variable, value)
     end
 
     def builder(verify_presence:)
-      yield if verify_presence.present?
+      yield if [verify_presence].flatten.reject(&:blank?).present?
```

Comment:
Can you think of a way for not parsing the input on the domain layer? Ideally `flatten` and `reject` could be done in the controller?

Reply:
This is a bit tricky. It's essentially defensive programming so that we're not relying on how `verify_presence` is provided to us.

I think this is necessary given the way the application is currently architected:

- `BaseController` constructs a `Filter` by passing whatever `params` values we have directly with no validation.
- `Filter` is a value object and sets no expectations about what a valid `Filter` attribute value is.
- `FindContent` sets no expectations about what a valid `Filter` is.
- `Content::Query` sets no expectations on the values provided to it's chain-able methods.

Reply:
`Filter` is currently validating the `sort_by` attribute, for example. A `ValueObject`, as any other object, can set a contract with their peers.  When we set any attribute on a Filter, we can/should transform it to something that is valid.